### PR TITLE
91788510 add generic lead event

### PR DESCRIPTION
**Pull request by lfender6445**
# [v2 tealium support](https://www.pivotaltracker.com/story/show/91788510)

collect values like listing id on lead success and  forward the data via events

### CODE REVIEW

File: jquery.leadservice.js [hunk 1]
```diff
@@ -221,18 +222,16 @@ define(['jquery', 'jquery.cookie'], function($) {
           data: $(this).serialize(),
           success: function(response) {
             status = 'success';
-            opts.lead_saved();
+            success = opts.lead_saved();
+            if (status == 'success') {
```
Comment (pasha) [inline jquery.leadservice.js]: looks like status is always will be `success`, do we need to keep checking for it?

Comment (lfender6445) [inline jquery.leadservice.js]: good point

File: jquery.leadservice.js [hunk 2]
```diff
@@ -220,19 +221,14 @@ define(['jquery', 'jquery.cookie'], function($) {
           type: 'POST',
           data: $(this).serialize(),
           success: function(response) {
-            status = 'success';
-            opts.lead_saved();
+            var data = opts.lead_saved();
+            $('body').trigger('lead_submission', (data || {}));
```
Comment (pasha) [inline jquery.leadservice.js:225]: not a big deal, it think this looks cleaner `var data = opts.lead_saved() || {};`

**Comment by pasha** [comment]
:+1:

File: jquery.leadservice.js (hunk 2 above)
Comment (lfender6445) [inline jquery.leadservice.js:225]: i don't particularly like variable declerations as arguments

**Pull request merged**